draft-ietf-quic-qpack.md

```diff
@@ -1001,17 +1010,11 @@ If the entry is in the static table, or in the dynamic table with an absolute
 index less than the Base, this representation starts with the '01' two-bit
 pattern.
 
-Only the header field name stored in the static or dynamic table is used. Any
-header field value MUST be ignored.
-
 The following bit, 'N', indicates whether an intermediary is permitted to add
 this header to the dynamic header table on subsequent hops. When the 'N' bit is
-set, the encoded header MUST always be encoded with a literal representation. In
-particular, when a peer sends a header field that it received represented as a
```

Comment:
I do not see any clarity or value that the "In particular.." sentence adds, it just rewords the previous sentence.  I propose to remove it.

Reply:
It was copied verbatim from HPACK.  Maybe we should leave it in.

Reply:
I see what you mean about it being a restatement, but it's a ramification of "MUST always" that isn't immediately obvious.  If it's not stated, people will mess this up, I guarantee.  (They do anyway, because not all HPACK libraries tell you whether this bit was set on the way in.)

It's not my favorite feature of HPACK, but let's either keep the clarification or remove the requirement entirely.  If you want to discuss the latter, I would suggest doing so in a separate issue and PR.